modules/end-user-guide/partials/proc_troubleshooting-slow-workspaces.adoc

```diff
@@ -34,7 +34,7 @@ Installing offline::
 +
 _Role: Administrator_
 +
-Components of {prod-short} are OCI images. Setup {prod} in offline mode (air-gap scenario) to allow for reducing any extra download at runtime as everything needs to be present from the beginning. See xref:installation-guide:installing-che-in-a-restricted-environment.adoc[].
+Components of {prod-short} are OCI images. Setup {prod} in offline mode (air-gap scenario) to reduce any extra download at runtime as everything needs to be available from the beginning. See xref:installation-guide:installing-che-in-a-restricted-environment.adoc[].
```

Comment:
```suggestion
Components of {prod-short} are OCI images. Set up {prod} in offline mode (air-gap scenario) to reduce any extra download at runtime because everything needs to be available from the beginning. See xref:installation-guide:installing-che-in-a-restricted-environment.adoc[].
```

Reply:
Noun: Setup
Verb: Set up
IBM SG:
```
as adverb
Do not use to mean "because" or "while".
Example: Write "Because you created the file, you can delete it", not "As you created the file, you can delete it".
```